datasets/san_francisco/pipelines/_images/run_csv_transform_kub/csv_transform.py

```diff
@@ -0,0 +1,552 @@
+import typing
+
+import pandas as pd
+import requests
+from google.api_core.exceptions import NotFound
+from google.cloud import bigquery, storage
+
+
+def main(
+    source_url: str,
+    pipeline_name: str,
+    source_file: pathlib.Path,
+    target_file: pathlib.Path,
+    project_id: str,
+    dataset_id: str,
+    table_id: str,
+    schema_path: str,
+    chunksize: str,
+    target_gcs_bucket: str,
+    target_gcs_path: str,
+    input_headers: typing.List[str],
+    rename_headers_list: typing.List[str],
+    empty_key_list: typing.List[str],
+    gen_location_list: dict,
+    resolve_datatypes_list: dict,
+    remove_paren_list: typing.List[str],
+    strip_newlines_list: typing.List[str],
+    strip_whitespace_list: typing.List[str],
+    date_format_list: typing.List[str],
+    reorder_headers_list: typing.List[str],
+) -> None:
+
+    logging.info(f"{pipeline_name} process started")
+    pathlib.Path("./files").mkdir(parents=True, exist_ok=True)
+    execute_pipeline(
+        source_url=source_url,
+        source_file=source_file,
+        target_file=target_file,
+        project_id=project_id,
+        dataset_id=dataset_id,
+        destination_table=table_id,
+        schema_path=schema_path,
+        chunksize=chunksize,
+        target_gcs_bucket=target_gcs_bucket,
+        target_gcs_path=target_gcs_path,
+        input_headers=input_headers,
+        rename_headers_list=rename_headers_list,
+        empty_key_list=empty_key_list,
+        gen_location_list=gen_location_list,
+        resolve_datatypes_list=resolve_datatypes_list,
+        remove_paren_list=remove_paren_list,
+        strip_newlines_list=strip_newlines_list,
+        strip_whitespace_list=strip_whitespace_list,
+        date_format_list=date_format_list,
+        reorder_headers_list=reorder_headers_list,
+    )
+    logging.info(f"{pipeline_name} process completed")
+
+
+def execute_pipeline(
+    source_url: str,
+    source_file: pathlib.Path,
+    target_file: pathlib.Path,
+    project_id: str,
+    dataset_id: str,
+    destination_table: str,
+    schema_path: str,
+    chunksize: str,
+    target_gcs_bucket: str,
+    target_gcs_path: str,
+    input_headers: typing.List[str],
+    rename_headers_list: typing.List[str],
+    empty_key_list: typing.List[str],
+    gen_location_list: dict,
+    resolve_datatypes_list: dict,
+    remove_paren_list: typing.List[str],
+    strip_newlines_list: typing.List[str],
+    strip_whitespace_list: typing.List[str],
+    date_format_list: typing.List[str],
+    reorder_headers_list: typing.List[str],
+) -> None:
+    if destination_table == "bikeshare_station_info":
+        source_url_json = f"{source_url}.json"
+        source_file_stations_json = (
+            str(source_file).replace(".csv", "") + "_stations.json"
+        )
+        download_file_json(
+            source_url_json, source_file_stations_json, source_file, "stations"
+        )
+    elif destination_table == "311_service_requests":
```

Comment:
Is there the possible of `destination_table` a whole different value, requiring an `else` statement here too?